spec/features/budgets/access_index.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+feature "access index" do
```

Comment:
I would rename this feature to be "user_can_see_only_their_budgets_spec"
And I would write it as so:

```rb
user = create(:user)
user_budget = create(:budget, user: user)
other_budget - create(:budget)
log_in_as user

visit budgets_path

expect(page).to have_text(I18n.t("budgets.index.title"))
expect(page).to have_text(budget.name)
expect(page).not_to have_text(other_budget.name)
```

Reply:
Thanks Elle, works a treat - sha: 1502d2e